ui/app/helpers/path-or-array.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
+
+export function pathOrArray([maybeArray, target]) {
+  if (Array.isArray(maybeArray)) {
+    return maybeArray;
+  }
+  return target.get(maybeArray);
+}
+
+export default Ember.Helper.helper(pathOrArray);
```

Comment:
hehe, this reminds me of something I've just done in one of my own PR's 🤔 make me think I should expand on what I've done myself - I was checking for a number, but I figure there it's relatively common to type check things in helpers. Anyway, just thinking out loud really

Reply:
Here's the link:

https://github.com/hashicorp/consul/pull/4677/files#diff-84d5eca0802aae16295fd808d3a95441

This is the 'possible beginnings' of something, something simple but looks like it might be useful, need to think about it a bit more tho. It's very specific to helpers, not sure whether to keep it that way or not at the moment. I half think its so simple its not wort hit, but then again with helpers it might be.